### PR TITLE
Code review

### PR DESCRIPTION
**Pull request by edgaryp**

### CODE REVIEW

File: src/store.js [hunk 1]
```diff
@@ -1,16 +1,74 @@
...
+        return tileClass.getTile();
+      });
+      const shuffleTilesInGrid = [];
+      for (let i = 0; i < state.puzzleDimension; i += 1) {
+        shuffleTilesInGrid.push(tiles.splice(0, state.puzzleDimension));
+      }
+      return shuffleTilesInGrid;
+    },
+    checkVictory(state) {
+      const tiles = state.tiles.slice(0, state.tiles.length - 1);
```
Comment (paulferrett) [inline src/store.js]: These are both the default values for slice(), so can be omitted.

File: README.md [hunk 2]
```diff
@@ -13,17 +13,4 @@ npm run serve
 ### Compiles and minifies for production
 ```
 npm run build
-```
-
-### Run your tests
-```
-npm run test
```
Comment (paulferrett) [inline README.md:20]: Do you reckon they're expecting tests though?

File: package.json [hunk 3]
```diff
@@ -32,7 +33,30 @@
       "plugin:vue/essential",
       "@vue/airbnb"
     ],
-    "rules": {},
+    "rules": {
+      "comma-dangle": [
+        "error",
+        "never"
```
Comment (paulferrett) [inline package.json]: I hate having no dangling commas :/

also judging by `babel.config.js`, they seem to like them too

File: src/store.js (hunk 1 above)
Comment (edgaryp) [reply src/store.js]: updated to `const tiles = state.tiles.slice(0, -1);`

File: package.json (hunk 3 above)
Comment (edgaryp) [reply package.json]: Enabled comma dangle

**Pull request merged**